libc/src/unistd/pathconf_utils.h

```diff
@@ -0,0 +1,24 @@
+//===-- Implementation header for pathconf_utils ----------------------*- C++
+//-*-===//
```

Comment:
nit: fix this license header

Reply:
sorry I dont quite get it, what exactly are you referring to ? remove the word "header" ?

Reply:
Could you make it in one line? (by deleting some `-`)